tests/e2e/operator-restart/assert-operator-pod.yaml

```diff
@@ -14,3 +14,32 @@ status:
     ready: true
     started: true
   phase: Running
+---
```

Comment:
Should we move this into `assert-operator-networkpolicy.yaml`?

Reply:
+1